test/e2e/app-dir/actions/app-action-size-limit-invalid.test.ts

```diff
@@ -0,0 +1,80 @@
+import { createNextDescribe } from 'e2e-utils'
+
+createNextDescribe(
```

Comment:
Can we test here configuring a higher value e.g. `2MB` and ensure a server action that sends a `1MB` request works fine but a `3MB` request fails correctly?